packages/react-app/src/contexts/BridgeContext.jsx

```diff
@@ -127,12 +128,19 @@ export const BridgeProvider = ({ children }) => {
   const transfer = useCallback(async () => {
     setLoading(true);
     try {
-      const tx = await relayTokens(
-        ethersProvider,
-        fromToken,
-        receiver || account,
-        fromAmount,
-      );
+      const tx = NATIVE_CURRENCY_SYBMOLS.includes(fromToken.symbol)
```

Comment:
the symbol check as mentioned in other places should not used. Use either `mode === NATIVE` check or some other custom param.
Also here that mode check can be done inside `relayTokens` itself, removing the logic from this method and taking it deeper inside.